addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,19 @@
+---Fields in module 'account_voucher'---
+account_voucher / account.voucher          / company_id (many2one)         : is now stored
+# DONE: post-migration: filled from account_journal
+
+account_voucher / account.voucher          / currency_id (many2one)        : is now stored
+# DONE: post-migration: compute the field
```

Comment:
No need of computing this again. When the field is not stored and gets stored, the compute is done on regular update process.